src/QuiqChatClient.js

```diff
@@ -47,9 +40,23 @@ class QuiqChatClient {
       HOST: this.host,
       CONTACT_POINT: this.contactPoint,
     });
+
+    window.addEventListener('online', () => {
```

Comment:
Nah.  This only fires when the user's network connectivity changes.  Presumably if they just loaded the webpage, they are online already, so this won't fire.  This is only to handle the case where they lose connection and then regain it.  In that case we will reconnect for them.